bin/parse_phonon.py

```diff
@@ -0,0 +1,401 @@
+
+displacement_dict = {
+    'natom': nAtoms['supercell'],
+    'first_atoms': []
+    }
+
+for i in IndAtoms['primitive']:
+    for label, dirVec in [['x', [1, 0, 0]], ['y', [0, 1, 0]], ['z', [0, 0, 1]]]:
+        for sl, s in [['+', +1], ['-', -1]]:
+            dirVec = np.array(dirVec) * s
+
+            outName = f'{arg.FrameworkName}_{i}_{sl}{label}'
+            forces = get_forces(outName, f'{i}_{sl}{label}')
+
+            displacement_dict['first_atoms'].append({
+                'number': i,
+                'displacement': arg.dR * dirVec,
+                'forces': forces
+                })
+
+# Displacement dict
+PhononCalc.dataset = displacement_dict
+
+# Set the force constants
+PhononCalc.produce_force_constants(calculate_full_force_constants=True)
+
+# Symmetrize the force constants
+PhononCalc.symmetrize_force_constants()
+
+# Hack to do irrep analysis
+ir_labels = [""] * nAtoms['primitive'] * 3
+PhononCalc.supercell.set_magnetic_moments(None)
+
+# Set IR for Gamma point
+PhononCalc.set_irreps([0.0, 0.0, 0.0])
+
+for i, (deg_set, ir) in enumerate(zip(PhononCalc.get_irreps()._degenerate_sets,
+                                      PhononCalc.get_irreps()._ir_labels)):
+    for j in deg_set:
+        if (ir is None):
+            ir_labels[j] = "Non"
+        else:
+            ir_labels[j] = ir
+
+# Get the dynamical matrix at Gamma
+dymMatrix_Gamma = PhononCalc.get_dynamical_matrix_at_q([0, 0, 0])
+
+# Calculate the eigenvalues (frequencies) and eigenvectors (normal displacements)
+eigenValues, eigenVectors = np.linalg.eigh(dymMatrix_Gamma)
+
+# Calculate sqrt(omega) keeping the sign of the eigenvalues and converting to cm-1
+frequencies = np.sqrt(np.abs(eigenValues.real)) * np.sign(eigenValues) * factor2cm
+
+# Check if there is any negative frequency
+if np.any(frequencies < -5e-3):
+    print('WARNING: Negative frequencies found!')
+
+if arg.SaveVecs:
+    shiftVecs = np.zeros((len(frequencies), nAtoms['primitive'], 3))
+    for i, mode in enumerate(eigenVectors.real.T):
+        for j, atom in enumerate(mode.reshape(-1, 3)):
+            shiftVecs[i, j] = atom / np.sqrt(atomMasses['primitive'][j])
+
+    os.makedirs(os.path.join(arg.output_folder, 'VIBRATION_FILES'), exist_ok=True)
+
+    # Save independend files for each mode
+    for i, freq in enumerate(frequencies):
+        save_axsf(os.path.join(arg.output_folder, 'VIBRATION_FILES'),
+                  f'{arg.FrameworkName}_{i}_{ir_labels[i]}_{freq}',
+                  cellMatrix['primitive'],
+                  atomTypes['primitive'],
+                  cartPos['primitive'],
+                  [shiftVecs[i]])
+
+    # Save all modes in a single file
+    save_axsf(arg.output_folder,
+              f'{arg.FrameworkName}_all',
+              cellMatrix['primitive'],
+              atomTypes['primitive'],
+              cartPos['primitive'],
+              shiftVecs)
+
+# Calculating Raman Tensor Placzek Invariants
```

Comment:
Are these two comments on "Raman Tensor Placzek Invariants" placeholders for a future feature, or are they higher-level comments that are then broken down in several pieces below?